requirements/testing_without_asyncio.txt

```diff
@@ -1,3 +1,3 @@
 # pip install -r requirements/testing_without_asyncio.txt
-pytest>=6.2.5,<8.4        # https://github.com/tornadoweb/tornado/issues/3375
+pytest>=6.2.5,<8.5        # https://github.com/tornadoweb/tornado/issues/3375
```

Comment:
🤔 Unsure if this version can remain unbound now that the linked issue is closed?

```suggestion
pytest>=6.2.5,<8.5
```